modules/RTC/ScreenObtainer.js

```diff
@@ -215,14 +215,31 @@ const ScreenObtainer = {
         }
 
         const { desktopSharingFrameRate } = this.options;
-        const video = typeof desktopSharingFrameRate === 'object' ? { frameRate: desktopSharingFrameRate } : true;
+        const setScreenSharingResolutionConstraints = browser.isChromiumBased()
+            && this.options?.testing?.setScreenSharingResolutionConstraints;
+        const video = {};
+
+        if (typeof desktopSharingFrameRate === 'object') {
+            video.frameRate = desktopSharingFrameRate;
+        }
+        if (setScreenSharingResolutionConstraints) {
+            // Set bogus resolution constraints to work around
+            // https://bugs.chromium.org/p/chromium/issues/detail?id=1056311
+            video.height = 99999;
+            video.width = 99999;
+        }
+
         const audio = this._getAudioConstraints();
 
         // At the time of this writing 'min' constraint for fps is not supported by getDisplayMedia.
         video.frameRate && delete video.frameRate.min;
 
+        if (Object.keys(video).length === 0) {
+            video = true;
```

Comment:
You best to make video a let now.

Reply:
Sorry, I'm using a new editor and not paying enough attention apparently 🤦